azext_iot/_help.py

```diff
@@ -1379,3 +1379,36 @@
         az iot dps compute-device-key -g {resource_group_name} --dps-name {dps_name}
         --enrollment-id {enrollment_id} --registration-id {registration_id}
 """
+
+helps[
+    "iot dps connection-string"
+] = """
+    type: group
+    short-summary: Manage IoT Hub Device Provisioning Service connection strings.
+"""
+
+helps[
+    "iot dps connection-string show"
+] = """
+    type: command
+    short-summary: Show the connection strings for the specified Device Provisioning Services using the given
+                   policy name and key.
+    examples:
+    - name: Show the connection strings for all active state DPS's in a subscription
```

Comment:
I think the grammar is off for `DPS's` since its describing plurality (doesn't need apostrophe). But `DPSs` isn't great.

We could spell out "Device Provisioning Services" as its done earlier, or perhaps saying "DPS instances", "DPS environments" etc.